src/javascript/JContent/actions/publishAction.jsx

```diff
@@ -62,7 +62,7 @@ function getButtonLabelParams(context, language, res, t) {
     }
 
     return {
-        displayName: t('jcontent:label.contentManager.selection.items', {count: context.pages.length}),
+        displayName: t('jcontent:label.contentManager.selection.items', {count: context.pages ? context.pages.length : 0}),
```

Comment:
What is context.pages ? Something happened at some point, but for me it should be context.paths

Reply:
Ahh, you are right of course, it was introduced by me when I moved it, not sue how I went from paths to pages given that I copied the code. My only explanation is that I work on router issue in parallel which got me thinking about pages.